x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -182,6 +185,90 @@ export const ModelDeploymentInProgress = ({
 export const ModelDeployed = ({
   dismiss,
   isDismissable,
+  isStartButtonDisabled,
+}: Pick<TextExpansionCallOutState, 'dismiss' | 'isDismissable' | 'isStartButtonDisabled'>) => {
+  const { startTextExpansionModel } = useActions(TextExpansionCalloutLogic);
+
+  return (
+    <EuiPanel color="success">
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexItem grow>
+          <EuiFlexGroup direction="row" gutterSize="s" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="checkInCircleFilled" />
+            </EuiFlexItem>
+            <EuiFlexItem grow>
+              <EuiTitle size="xs">
+                <h4>
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.deployedTitle"
+                    defaultMessage="Your ELSER model has deployed but not started." />
+                </h4>
+              </EuiTitle>
+            </EuiFlexItem>
+            {isDismissable && (
+              <EuiFlexItem grow={false}>
+                <TextExpansionDismissButton dismiss={dismiss} />
+              </EuiFlexItem>
+            )}
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem grow>
+          <EuiText>
+            <FormattedMessage
+              id="xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.deployedBody"
+              defaultMessage="You may start the model in a single-threaded configuration for testing, or tune the performance for a production environment." />
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow>
+          <EuiFlexGroup direction="row" gutterSize="s" alignItems="center">
+            <EuiFlexItem>
+              <EuiButton
+                aria-label={i18n.translate(
```

Comment:
When this component loads, I get the following warning in the console:
```
iButtonIcon requires aria-label or aria-labelledby to be specified because icon-only
      buttons are screen-reader-inaccessible without them. 
    at EuiButtonIcon (http://localhost:5601/cqx/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:80995:25)
    at span
    at http://localhost:5601/cqx/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:170434:73
    at http://localhost:5601/cqx/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:156939:22
    at EuiToolTip (http://localhost:5601/cqx/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:156558:81)
    at div
    at http://localhost:5601/cqx/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:170434:73
    at EuiFlexItem (http://localhost:5601/cqx/9007199254740991/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:109977:23)
    at div
   ...
 ```

Not sure why; these are the only two buttons rendered and they both have `aria-label`. Is `ButtonIcon` a sub-component of one of these buttons?

Reply:
If you look at the stacktrace, this is happening inside an `EuiToolTip` element. I tracked it down to this element, which we apparently missed adding a label to seven months ago. If you can fix that in this PR that'd be great, otherwise I'll create a separate PR:

https://github.com/elastic/kibana/blob/ce71264d88c46c45ed8ac00dc6c74bcbd05809c7/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx#LL169C9-L169C9

Also, you should remove the aria labels from these two normal`EuiButton` elements. Aria-labels are only needed for buttons that don't have labels, or when you really need to add extra context for folks accessing the page with a screen reader. Having different labels in an aria-label and the actual label of the button is also bad practice: screen reader users and visual users should have the same experience.

Reply:
Added a fix in this PR: https://github.com/elastic/kibana/pull/156080/files#diff-b76b853a1c67155ed6f0c2134f4fcbd2514597814e2936ce1625f011b8b3b3cd